docs/docs/guides/labs/dg/multiple-projects.md

```diff
@@ -17,42 +17,40 @@ If you're just getting started, we recommend [scaffolding a single project](/gui
 
 If you need to collaborate with multiple teams, or work with conflicting dependencies that require isolation from each other, you can scaffold a workspace directory that contains multiple projects, each with their own separate Python environment.
```

Comment:
I think it is worth noting more advantages of having a workspace, like easily being able to view all of them in a single webserver, and (eventually) making unified CI/CD better